MinPlatformPkg/Test/Library/TestPointCheckLib/StandaloneMmTestPointCheckLib.inf

```diff
@@ -0,0 +1,59 @@
+## @file
+# Component information file for Board Init Test Library
+#
+# Copyright (c) 2017-2019, Intel Corporation. All rights reserved.<BR>
+#
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+##
+
+[Defines]
+  INF_VERSION                    = 0x00010005
+  BASE_NAME                      = StandaloneMmTestPointCheckLib
+  FILE_GUID                      = AE975D85-61EA-48F1-A68B-BA777751AC21
+  MODULE_TYPE                    = MM_STANDALONE
+  VERSION_STRING                 = 1.0
+  PI_SPECIFICATION_VERSION       = 0x00010032
+  LIBRARY_CLASS                  = TestPointCheckLib|MM_STANDALONE DXE_SMM_DRIVER
```

Comment:
I think you can drop `DXE_SMM_DRIVER` here. This library instance is specifically written for Standalone MM, right?